### DISCUSSION TITLE
Support RockPro64?

### DISCUSSION THREAD

**Discussion by sfxworks** [ltsp/ltsp#511] [Ideas]
I see it's easy to specify a ltsp image for the raspberry pi. Can the same be done for this arm board?

**Comment by alkisg** [comment]
I don't think any changes will be needed in LTSP itself.
But someone would need to write a tutorial on how to configure u-boot on the clients for netbooting.

https://wiki.pine64.org/wiki/ROCKPro64#Booting_from_USB_or_PXE
